File: TaskManager/TaskManager_210042112/TaskManager_112/TaskManager_112/Issues/IssueRepository.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TaskManager_112
{
    public class IssueRepository
    {
        Dictionary<int, Issue> issues = new Dictionary<int, Issue>();

        public void CreateIssue(Issue issue)
        {
            issues.Add(issue.ID, issue);
        }

        public Issue GetIssue(int id)
        {
            return issues.ContainsKey(id) ? issues[id] : null; // does the dictionary contain the id or not
        }
    }
}

File: TaskManager/Lab_Final/src/main/java/IssueRepository.java
import java.util.HashMap;
import java.util.Map;

public class IssueRepository {

    private final Map<Integer, Issue> issues = new HashMap<>();

    public void addIssue(Issue issue) {

        issues.put(issue.getId(), issue);
    }
    public Issue getIssue(int id){
        return issues.get(id);
    }

}